ci/scripts/start_instance.sh

```diff
@@ -74,6 +74,8 @@ while true; do
     LABELS="instance_type=heavy-lifter,time-to-live=${TTL},job-name=${SANITIZED_BUILD_JOB_NAME},pipeline-name=${SANITIZED_BUILD_PIPELINE_NAME},build-name=${SANITIZED_BUILD_NAME}"
 
     set +e
+# Try to kill any existing machine before starting
+    gcloud compute --project=${PROJECT} instances delete ${INSTANCE_NAME} --zone=${ZONE} --delete-disks=all -q
```

Comment:
So is it the case that even though the create failed something is "created" that keeps the retry from working?

Reply:
I see that the first task timed out after creating the instance, so yes deleting the instance should correct that.